lib/tests/streamlit/app_session_test.py

```diff
@@ -39,86 +39,6 @@ def del_path(monkeypatch):
 
 
 class AppSessionTest(unittest.TestCase):
-    @patch("streamlit.app_session.config")
-    @patch("streamlit.app_session.SessionData")
-    @patch("streamlit.app_session.LocalSourcesWatcher")
-    def test_enqueue_without_tracer(self, _1, _2, patched_config):
```

Comment:
AppSession no longer calls "maybe_handle_execution_control_request". These two tests have been replaced with a single parameterized test in `ScriptRunnerTest`, below.